src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly

```diff
@@ -92,6 +92,9 @@
                                     <f:entry title="${%Docker Image}" field="dockerImage" description="If using Docker, specify the docker image.">
                                         <f:textbox field="dockerImage" clazz="required" value="${slaveInfo.containerInfo.dockerImage}" default=""/>
                                     </f:entry>
+                                    <f:entry title="${%Docker Privileged Mode}" field="dockerPrivilegedMode" description="This will start the image using Docker's privileged mode.">
+                                        <f:checkbox field="dockerPrivilegedMode" value="${slaveInfo.containerInfo.dockerPrivilegedMode}" checked="false" />
```

Comment:
Open config, set checkbox, save, check xml config on fs, then open this page again and review checkbox (in case of wrong fatabinding). Also, why checked is need to be defined at all? There is field, there is default option and there is checkbox default itself.